tests/e2e/cucumber/features/smoke/denyShareAccess.feature

```diff
@@ -0,0 +1,74 @@
+Feature: deny share access
```

Comment:
@ScharfViktor could you please help out and review the e2e part of this PR? thank you :-)